docs/release-notes/highlights-7.4.0.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[release-highlights-7.4.0]]
+== 7.4.0 release highlights
+++++
+<titleabbrev>7.4.0</titleabbrev>
+++++
+
+Each release of {kib} brings new features and product improvements. 
+Here are the most notable feature and user experience changes in 7.4.
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+See the {kib} 7.4 Release Blog for a complete list of highlights.
+
+[float]
+[[pki-7.4]]
+=== Public Key Infrastructure (PKI) authentication
+
+Building off the success of features such as feature controls in 7.2 and 
+support for Kerberos in 7.3, {kib} 7.4 adds support for another 
+highly requested authentication mechanism, Public Key Infrastructure (PKI). 
+With native support for PKI authentication, users are now able to log into 
+{kib} using X.509 client certificates and a two-way encryption system. 
+This provides new ways to authenticate {kib}--from general client certificate 
+authorization to identity card access. To start using PKI in production, check out 
+<<pki-authentication, our documentation>>.
+
+[float]
+[[siem-gets-maps-7.4]]
+=== SIEM gets maps
+
+A geospatial perspective is especially useful for data sets for security 
```

Comment:
How about `security event data sets` instead of `for data sets for security events`?